test/src/tests-store.ts

```diff
@@ -43,7 +46,7 @@ describe('Store', () => {
         expect(store.isFirstRun, 'isFirstRun').to.eq(null);
         expect(store.clientId, 'clientId').to.eq(null);
         expect(store.deviceId, 'deviceId').to.eq(null);
-        expect(store.devToken, 'devToken').to.eq(null);
+        expect(store.devToken, 'devToken').to.eq(apiKey);
```

Comment:
If `apiKey` is optional, this should be null and the following test on line 87 tests for the presence of `apiKey`.